plugins/hydra_nevergrad_sweeper/setup.py

```diff
@@ -20,6 +20,10 @@
             "Programming Language :: Python :: 3.8",
             "Operating System :: OS Independent",
         ],
-        install_requires=["hydra-core>=1.0.0rc1", "nevergrad>=0.3.2"],
+        install_requires=[
+            "hydra-core>=1.0.0rc1",
+            # TODO move to v0.4.0 when ready (before merge!)
+            "nevergrad@git+git://github.com/facebookresearch/nevergrad.git@master#egg=isort",
+        ],
```

Comment:
likestamp.
What are you relying on from master that is not in the last release?

Reply:
The "lower" and "upper" arguments of Scalar were added after 0.3.2

Reply:
I see, so before that the official bounds is +/- inf?

Reply:
it's a bit more complicated, if you dont specify bounds it is indeed, but specifying **both** bounds is a bit special since it gives you some kind of prior on the variable, so the step size can be adapted to it, otherwise you need to specify a step size yourself